src/utils/renderPDF.test.ts

```diff
@@ -0,0 +1,29 @@
+import renderPDF from './renderPDF'
+import readStream from './readStream'
+import mockOf from '../../test/utils/mockOf'
+
+jest.mock('./readStream')
+
+const readStreamMock = mockOf(readStream)
+
+test('renders simple PDF documents', async () => {
+  readStreamMock.mockResolvedValueOnce(Buffer.of(1, 2, 3))
```

Comment:
nit: should `Buffer.of(1,2,3)` be specified once and then reused, rather than repeatng yourself?

Reply:
In tests I tend to repeat values if the connection is fairly obvious. I don’t feel too strongly about it.